index.bs

```diff
@@ -1719,6 +1719,56 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+New web features should be self-contained and self-describing and ideally should not require an additional manifest file.
+Manifest files can add complexity to the development and deployment process for web technologies.  Sometimes manifest 
+files are required, but the use of different manifests on the web should be kept to a minimum.
+
+You may need to make a new manifest file if the domain of the manifest file is different from the existing manifest files.
+For example, if the fetch timing is different, or if the complexity of the manifest warrants it.  Application metadata should be added 
+to the Web App Manifest or be an extension of it. 
+Manifests designated to be used for specific applications or which require interoperability with non-browsers may need to take a different approach.
+Payment Method Manifest, Publication Manifest, and Origin Policy are examples of these cases.
+
+For example, if you have a single piece of metadata, even if the fetch timing is different than an existing manifest,
+it is probably best to use an existing manifest (or ideally design the feature in such a way that a manifest is not required).
+However, if your feature requires a complex set of metadata specific to a functional domain, the creation of a new manifest may be justified.
+
+Note that in all cases, the naming conventions should be harmonized (see section 2.2)
+ 
+Note: By principle, existing manifests use lowercase, underscore-delimited names.
```

Comment:
So let's just make it this then @domenic ?

Note: By principle, existing manifest keys use lowercase, underscore-delimited names. If a 
key can clearly be expressed as a single word, that is recommended.

Reply:
Seems reasonable, perhaps with a link to https://w3ctag.github.io/design-principles/#casing-rules